docs/proposals/metrics-api.md

```diff
@@ -0,0 +1,134 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
```

Comment:
It would be easier to review the proposal as a new one. There is a reference to the previous one in the first line.